src/server/cmd/pachd/main.go

```diff
@@ -864,8 +884,10 @@ func doFullMode(config interface{}) (retErr error) {
 			return err
 		}
 		if err := logGRPCServerSetup("Enterprise API", func() error {
+			e := eprsserver.EnvFromServiceEnv(env, path.Join(env.Config().EtcdPrefix, env.Config().EnterpriseEtcdPrefix), txnEnv)
+			e.Mode = eprsserver.UnpausedMode
```

Comment:
can we make the default value for `e.Mode = eprsserver.UnpausedMode` so that we don't have to explicitly set it besides in paused mode?

Reply:
The zero value is unpaused, so this works, but I think that explicit is probably better than implicit. I can kinda see both sides though.

Reply:
I think we should make paused opt-in for future modes, so if we could avoid forcing all modes to implement the "base logic" for the feature, I think that would be preferable. WDYT?

Reply:
Hmmm, I could change `server.PauseMode` to enumerate pausable modes (+paused mode) instead of all possible modes, and have the zero value be `UnpausableMode`. Then `Pause` and `Unpause` could key off of that.

The end result would be that one need not implement anything for a non-pausable mode.